src/repos/UserRepo.ts

```diff
@@ -65,6 +66,37 @@ const unfollowPublication = async (uuid: string, pubSlug: string): Promise<User>
   return user;
 };
 
+/**
+ * Adds organization slug to user's followedOrganizations.
+ * Requires: the user is not already following the organization.
+ */
+const followOrganization = async (uuid: string, slug: string): Promise<User> => {
+  const user = await UserModel.findOne({ uuid });
+
+  if (user) {
+    user.followedOrganizations.push(new OrganizationSlug(slug));
+    return user.save();
+  }
+  return user;
+};
+
+/**
+ * Deletes organization slug from user's followedOrganizations.
+ */
+const unfollowOrganization = async (uuid: string, orgSlug: string): Promise<User> => {
+  const user = await UserModel.findOne({ uuid });
+  if (user) {
+    const orgSlugs = user.followedOrganizations.map((orgSlugObj) => orgSlugObj.slug);
+    const orgIndex = orgSlugs.indexOf(orgSlug);
+
+    if (orgIndex === -1) return user;
+
+    user.followedOrganizations.splice(orgIndex, 1);
+    return user.save();
+  }
```

Comment:
same here

Reply:
should be checking if user is null or not